### ISSUE TITLE
Can I do this on Server 2016?

### ISSUE THREAD

**Issue by spottedmahn** [MicrosoftDocs/Virtualization-Documentation#1027]
The first few paragraphs mention doing this on Server 2016 but then the prerequisites say:

&gt;One computer system (physical or virtual) running Windows Server 2019

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 44a44120-01fd-11ca-9d51-21ac35609f11
* Version Independent ID: 75a53123-64be-1caf-fd67-7e7bef3b0aa0
* Content: [Windows Containers on Windows Server](https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/quick-start-windows-server)
* Content Source: [virtualization/windowscontainers/quick-start/quick-start-windows-server.md](https://github.com/Microsoft/Virtualization-Documentation/blob/master/virtualization/windowscontainers/quick-start/quick-start-windows-server.md)
* Service: **windows-containers**
* Product: **windows-containers**
* GitHub Login: @cwilhit

**Comment by olljanat** [comment]
@spottedmahn yes you can but my opinion is that go directly to Win Srv 2019 with latest docker version and all win updates (especially April update).

Win Srv 2016 a lot on weaknesses which will not ever get fixed to it.
